README.md

```diff
@@ -1,124 +1,198 @@
 
-## How To Run The Sample
+## About the code
 
-To run this sample you will need:
+The code for handling the token acquisition process is simple, as it boils down to:
+
+- one call for obtaining the code to display to the user (via `AcquireDeviceCodeAsync`)
+- and one call to poll the service to retrieve the tokens when available (via `AcquireTokenByDeviceCodeAsync`).
+
+You can find both calls in the sample in the static method `GetTokenViaCode`, from the app root class `Program` in program.cs.
+
+## How to run this sample
+
+To run this sample, you'll need:
+
+- [Visual Studio 2017](https://aka.ms/vsdownload) or just the [.NET Core SDK](https://www.microsoft.com/net/learn/get-started)
 - An Internet connection
 - A Windows machine (necessary if you want to run the app on Windows)
 - An OS X machine (necessary if you want to run the app on OSX)
 - A Linux machine (necessary if you want to run the app on OSX)
-- Visual Studio 2015 (recommended) or DNX command line tools
-- An Azure Active Directory (Azure AD) tenant. For more information on how to get an Azure AD tenant, please see [How to get an Azure AD tenant](https://azure.microsoft.com/en-us/documentation/articles/active-directory-howto-tenant/) 
-- A user account in your Azure AD tenant. This sample will not work with a Microsoft account, so if you signed in to the Azure portal with a Microsoft account and have never created a user account in your directory before, you need to do that now.
+- An Azure Active Directory (Azure AD) tenant. For more information on how to get an Azure AD tenant, please see [How to get an Azure AD tenant](https://azure.microsoft.com/en-us/documentation/articles/active-directory-howto-tenant/)
+- A user account in your Azure AD tenant. This sample will not work with a Microsoft account. If you signed in to the Azure portal with a Microsoft account (previously named live account) and have never created a user account in your directory before, you need to do that now.
 
-### Step 1: Setup .NET core
+### Step 1: Clone or download this repository
 
-The ASP.NET documentation pages provide step by step instructions for installing .NET Core (the .NET Execution Environment) for your platform of choice.
-#### Windows
-If you are targeting Windows, see [this document](https://docs.asp.net/en/latest/getting-started/installing-on-windows.html).
-(this readme assumes you will follow the instructions for installing ASP.NET 5 with Visual Studio, but you can also go for the standalone option).
-#### Mac OS X
-If you are targeting Mac OS X, please follow the instructions [here](https://docs.asp.net/en/latest/getting-started/installing-on-mac.html). 
-#### Linux
-If you are targeting Linux, please follow the instructions [here](https://docs.asp.net/en/latest/getting-started/installing-on-linux.html). This sample has been tested on Ubuntu 
+Once you have completed the DNX setup, from your shell or command-line run:

```

Comment:
>DNX [](start = 28, length = 3)

I think this terminology should be explained. e.g. is that all the pre-reqs above or something else?

Reply:
Good catch.